dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -1416,6 +1417,7 @@ def test_base(self, *mocks):
         self.assertEqual(4, profiler.total_samples)
         self.assertCountEqual(['this', 'is my', 'test'], profiler.sample)
         self.assertEqual(1, profiler._empty_line_count)
+        self.assertAlmostEqual(1.43051e-05, profiler.capacity)
```

Comment:
do these need to be almost equals? Don't we know what the exact value should be?
e.g. `15/1024**2`

Reply:
changed